molgenis-api-metadata/src/main/java/org/molgenis/api/metadata/v3/MetadataApiJobService.java

```diff
@@ -26,20 +26,16 @@
   /**
    * Schedule an entity type delete job.
    *
-   * @param entityTypeId entity type to delete
+   * @param entityType deleted entity type
```

Comment:
I prefer the current documentation ```entity type to delete``` since the entity type hasn't been deleted yet.